src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -638,6 +638,13 @@ def get_organization_owner_members(
 
         return list(map(serialize_member, owner_members))
 
+    def get_organization_members_summaries(
+        self, *, organization_id: int
+    ) -> list[RpcOrganizationMemberSummary]:
+        org_members = OrganizationMember.objects.filter(organization_id=organization_id)
+
+        return [summarize_member(member) for member in org_members]
+
```

Comment:
Do you think we need to worry about pagination here, or the data size will always be something manageable? Also for orgs that don't exist or are bad values, should we return a 400 or 404 instead of an empty list?

Reply:
There are a number of orgs with many thousands of members. Are you sure you need them all?

In a control silo context you can also use `OrganizationMemberMapping` to read the basic membership information. You could have a non-RPC method that contains alternate implementations based on the silo mode.

We have an example of this in `sentry.services.hybrid_cloud.organization._organization_check_service`. Where it has alternate implementation for control and region modes. That would avoid serializing thousands of records across a network request.

Reply:
oh i didn't realize `OrganizationMemberMapping` was on the control silo. i'll use that instead since it has user information, thank you!